sdks/node-sdk/package.json

```diff
@@ -53,7 +53,7 @@
     "@xmtp/content-type-group-updated": "^1.0.1",
     "@xmtp/content-type-primitives": "^1.0.3",
     "@xmtp/content-type-text": "^1.0.1",
-    "@xmtp/node-bindings": "^0.0.22",
```

Comment:
if you bump to `0.0.25` you can get the release into here I just pushed too, to fix streams

Reply:
Sure thing